keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -3117,6 +2960,25 @@ already received all the relevant updates (Pod, ResourceClaim, etc.).
   - Testing: An E2E test covers the expected retry mechanism in kubelet when
     `NodePrepareResources` fails intermittently.
 
+- A DRA driver prepares a ResourceClaim for a pod, then fails or gets uninstalled.
+  The driver is needed to tear down the Pod.
+
+  - Detection: A Pod does not reach a final phase ("Succeeded" or "Failed") and
+    cannot be deleted. The `dra_resource_claims_in_use` for the node and driver
+    is larger than zero. The `dra_operations_seconds` metric for
+    `method_name: NodeUnprepareResources` with non-zero `grpc_status_code` increases.
+
+  - Mitigations: driver failure scenarios and mitigations must be documented by
+    the vendor. If it got deinstalled, it needs to be reinstalled.
+
+    With [device taints](https://github.com/kubernetes/enhancements/issues/5055),
+    admins can be sure that a driver is not needed anymore before removing it:
+    - Taint devices of a driver, optionally evicting pods.
+    - Check `dra_resource_claims_in_use`.
+    - When zero, remove the driver.
+
+  - Testing: An E2E test covers the expected retry mechanism in kubelet when
+    `NodeUnprepareResources` fails intermittently.
```

Comment:
I think a reference to a separate feature is useful here and does not make that feature a blocker for GA because it just improves how admins deal with this, it's not required.

Reply:
To clarify - when you say "a separate feature" and "that feature" here, are you referring to the deferred ideas (as opposed to what we hoisted for 1.34 reliability) discussed in context of https://github.com/kubernetes/kubernetes/issues/125542

Reply:
Here "separate feature" = "device taints and tolerations", which is available in 1.34 as alpha, not those two issues, which aren't.